dxil_spirv_c.h

```diff
@@ -363,6 +363,7 @@ typedef enum dxil_spv_option
 	DXIL_SPV_OPTION_SUBGROUP_PROPERTIES = 34,
 	DXIL_SPV_OPTION_DESCRIPTOR_HEAP_ROBUSTNESS = 35,
 	DXIL_SPV_OPTION_COMPUTE_SHADER_DERIVATIVES_NV = 36,
+	DXIL_SPV_OPTION_QUAD_CONTROL_RECONVERGENCE = 37,
```

Comment:
Need to update `DXIL_SPV_API_VERSION_MINOR` as well as the minor version in CMakeLists.txt.